HCK15_2023_Janelia/README.md

```diff
@@ -0,0 +1,142 @@
+
+**Directions:** The Janelia Research Campus is located in Loudoun County, Virginia – just 30 miles from Washington, DC, and about eight miles north of Dulles International Airport (IAD). For directions see [https://www.janelia.org/directions](https://www.janelia.org/directions)
+
+
+## Organizing Committee
+
+**Site chair:** [Jakob Voigt](https://www.voigtslab.org/people)
+
+**Program chair (User Days):**  [Oliver Rübel](https://crd.lbl.gov/divisions/scidata/mla/staff/oliver-ruebel/) and [Ben Dichter](http://bendichter.com/)
+
+**Program chair (Developer Days):** [Oliver Rübel](https://crd.lbl.gov/divisions/scidata/mla/staff/oliver-ruebel/) and [Ryan Ly](https://crd.lbl.gov/divisions/scidata/mla/staff/ryan-ly/)
+
+**Administrative Support:** [Janine Stevens](https://www.janelia.org/people/janine-stevens)
+
+### Additional Organizational Support
+
+- The Kavli Foundation
+- Janelia HHMI
+
+## What to bring?
+
+* Bring any example data sets needed for your project with you to the hackathon. For any lab-specific data (i.e., 
+  data not in NWB), you should know how to read the data using Python or MATLAB and ideally have scripts ready for 
+  reading the data.
+* Bring your laptop with appropriate software installed. For installation instructions see:
+  * [**PyNWB (Python)**](https://pynwb.readthedocs.io/en/stable/install_users.html). If you are a developer and want to contribute to PyNWB see the [**Installing PyNWB for Developers**](https://pynwb.readthedocs.io/en/stable/install_developers.html) instructions. 
+  * [**MatNWB (Matlab)**](https://neurodatawithoutborders.github.io/matnwb/)
+* For an overview of NWB software, see also: 
+  * [**Glossary of Core NWB Tools**](https://nwb-overview.readthedocs.io/en/latest/core_tools/core_tools_home.html) 
+  * [**Glossary of Analysis and Visualization Tools**](https://nwb-overview.readthedocs.io/en/latest/tools/tools_home.html)
+* For an overview of NWB see the [**NWB Overview Docs**](https://nwb-overview.readthedocs.io)
+
+## Objective
+
+The [Neurodata Without Borders](nwb.org) project is an effort to standardize the description and storage of neurophysiology
+data and metadata. NWB enables data sharing and reuse and reduces the energy barrier to applying data analytics both within
+and across labs. NWB has seen wide adoption in the neurophysiology community, and there are now over 100 datasets on the
+DANDI Archive in NWB, including data from the Allen Institute and the International Brain Laboratory.
+
+The **User Days** will train users how to convert their data to NWB and publish it on the DANDI Archive. 
+We will work with members of the neuroscience community that want to apply NWB to their datasets. We will train 
+attendees, starting from the basics and proceeding to advanced data engineering techniques to maximally utilize the
+features of the HDF5 and Zarr backends. Attendees will also be trained in the creation of NWB extensions.
+
+<<<<<<< HEAD:HCK15_16_2023_Janelia/README.md
```

Comment:
```suggestion
```

Reply:
Oops, that was probably my fault. I ran into merge conflicts and probably forgot to remove that line. Thanks for the fix.